### ISSUE TITLE
Allow to specify region in check_records

### ISSUE THREAD

**Issue by SAuroux** [cubing/worldcubeassociation.org#621]
Currently, there is no decent way to check if records are affected if the nationality of a competitor needs to be fixed. Being able to specify the region (e.g. "Germany" --> just look for German NRs) in this script would be a great support for the results team.

**Comment by jfly** [comment]
Just checking, doesn't changing the nationality of a competitor potentially affect 2 countries? Both the old and the new?

**Comment by SAuroux** [comment]
Yes, both would need to be checked then. Currently, we would potentially
need to select every competition of the competitor individually.
Am 17.05.2016 04:38 schrieb "Jeremy Fleischman" <notifications@github.com>:
